crates/bevy_input/src/state.rs

```diff
@@ -0,0 +1,32 @@
+/// The current state of an element.
+///
+/// ## Usage
+///
+/// It is used to define the state of the [`KeyboardInput`](crate::keyboard::KeyboardInput) and
+/// [`MouseButtonInput`](crate::mouse::MouseButtonInput).
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+pub enum ElementState {
```

Comment:
This is probably better called "ButtonState": all inputs that can be pressed are button-like. Element is a very generic term.